charts/rapid-response/ci/test-values.yaml.template

```diff
@@ -0,0 +1,5 @@
+sysdig:
+  accessKey: ${SECURE_AGENT_TOKEN}
+rapidResponse:
+  apiEndpoint: secure.sysdig.com
```

Comment:
This would already be handled by the helper, right?

Reply:
Yep this would already be handled by the helper, I have also test it by using the global parameter and the parameter `rapidResponse.apiEndpoint`, it works with both

```
{{/*
Determine collector endpoint based on provided region or .Values.rapidResponse.apiEndpoint
*/}}
{{- define "rapidResponse.apiEndpoint" -}}
    {{- if .Values.rapidResponse.apiEndpoint -}}
        {{- .Values.rapidResponse.apiEndpoint -}}
    {{- else if (eq .Values.global.sysdig.region "us1") -}}
        {{- "secure.sysdig.com" -}}
    {{- else if (eq .Values.global.sysdig.region "us2") -}}
        {{- "us2.app.sysdig.com" -}}
    {{- else if (eq .Values.global.sysdig.region "us3") -}}
        {{- "app.us3.sysdig.com" -}}
    {{- else if (eq .Values.global.sysdig.region "us4") -}}
        {{- "app.us4.sysdig.com" -}}
    {{- else if (eq .Values.global.sysdig.region "eu1") -}}
        {{- "eu1.app.sysdig.com" -}}
    {{- else if (eq .Values.global.sysdig.region "au1") -}}
        {{- "app.au1.sysdig.com" -}}
    {{- end -}}
{{- end -}}
```